_sass/_layout.scss

```diff
@@ -372,3 +372,32 @@ q:after {
 .downloads-content p:first-child{
   color: red;
 }
+
+/**
+  * Style for Semba Page, Conformal Mesher, OpenSEMBA, Ugrfdtd and zMesher 
+*/
+.semba-content,
+.conformal-mesher-content,
+.opensemba-content,
+.ugrfdtd-content,
+.zmesher-content{
+  width: 85%;
+  margin: auto;
+  text-align: justify;
+  a{
+    border-bottom: none;
+  }
+  ul,ol{
+    margin-left: 7%;
+  }
+}
+/**
+  * Style for Conformal Mesher, OpenSEMBA, Ugrfdtd and zMesher lists
+*/
+.conformal-mesher-content,
```

Comment:
Is this really needed?

Reply:
Not really, but it looks more similar now to the original page, it looks more aesthetic with the margin

Reply:
I'm saying it because this case is already covered in the previous one for those classes:
```
ul,ol{
    margin-left: 7%;
}
```